evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -40,465 +40,204 @@
-                    [DropNullColumns, Imputer] + delayed_features + estimator_components
-                )
-            pipeline.component_graph.compute_order == [
-                component.name for component in expected_components
-            ]
+    return _get_test_data_from_configuration
 
 
+@pytest.mark.parametrize("lognormal_distribution", [True, False])
 @pytest.mark.parametrize("input_type", ["pd", "ww"])
 @pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
-def test_make_pipeline_text_columns(input_type, problem_type):
-    X = pd.DataFrame(
-        {
-            "numerical": [1, 2, 3, 1, 2],
-            "categorical": ["a", "b", "a", "c", "c"],
-            "text": [
-                "string one",
-                "another",
-                "text for a column, this should be a text column!!",
-                "text string",
-                "hello world",
-            ],
-        }
-    )
-    y = pd.Series([0, 0, 1, 1, 0])
-    if input_type == "ww":
-        X.ww.init()
-        y = ww.init_series(y)
-    estimators = get_estimators(problem_type=problem_type)
-
-    pipeline_class = _get_pipeline_base_class(problem_type)
-    if problem_type == ProblemTypes.MULTICLASS:
-        y = pd.Series([0, 2, 1, 2])
-
-    for estimator_class in estimators:
-        if problem_type in estimator_class.supported_problem_types:
-            parameters = {}
-            if is_time_series(problem_type):
-                parameters = {
-                    "pipeline": {"date_index": None, "gap": 1, "max_delay": 1},
-                    "Time Series Baseline Estimator": {
-                        "date_index": None,
-                        "gap": 1,
-                        "max_delay": 1,
-                    },
-                }
-
-            pipeline = make_pipeline(X, y, estimator_class, problem_type, parameters)
-            assert isinstance(pipeline, pipeline_class)
-            delayed_features = []
-            if is_time_series(problem_type):
-                delayed_features = [DelayedFeatureTransformer]
-            if estimator_class.model_family == ModelFamily.LINEAR_MODEL:
-                estimator_components = [OneHotEncoder, StandardScaler, estimator_class]
-            elif estimator_class.model_family == ModelFamily.CATBOOST:
-                estimator_components = [estimator_class]
-            else:
-                estimator_components = [OneHotEncoder, estimator_class]
-            if estimator_class.model_family == ModelFamily.ARIMA:
-                expected_components = [TextFeaturizer, Imputer] + estimator_components
-            else:
-                expected_components = (
-                    [TextFeaturizer, Imputer] + delayed_features + estimator_components
-                )
-            pipeline.component_graph.compute_order == [
-                component.name for component in expected_components
-            ]
-
+@pytest.mark.parametrize(
+    "test_description, column_names",
```

Comment:
is `test_description` just to allow people who look at the test to understand what each test is param iteration is testing?

Reply:
Yup! Now that we can't use the test names to understand, I thought having `test_description` would make it easier to see at a quick glance what each test case is for 😁